sms/v1/openapi.yaml

```diff
@@ -525,86 +532,78 @@ paths:
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/ContactArrayResponse"
-            "*/*":
-              schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/ContactArrayResponse"
+                $ref: "#/components/schemas/ContactArrayResponse"
         "204":
           description: Empty list; no contacts were found for this friendly name.
+        "401":
+          description: Unauthorized
           content:
             application/json:
               schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/ContactArrayResponse"
-            "*/*":
-              schema:
-                type: array
-                items:
-                  $ref: "#/components/schemas/ContactArrayResponse"
-        "401":
-          description: Unauthorized
+                $ref: "#/components/schemas/ErrorResponse"
         "403":
           description: Forbidden
+          content:
+            application/json:
+              schema:
+                $ref: "#/components/schemas/ErrorResponse"
         "404":
           description: Not Found
-        "500":
-          description: Something went wrong. :-(
           content:
             application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
-            "*/*":
+        "500":
+          description: Something went wrong. :-(
+          content:
+            application/json:
               schema:
                 $ref: "#/components/schemas/ErrorResponse"
     post:
       tags:
         - BYON Contact Service
       summary: Create a contact
       description: Creates a new contact.
+      security:
+        - ApiKeyHeader: []
       operationId: createContactUsingPOST
       requestBody:
         $ref: "#/components/requestBodies/ContactEntity"
```

Comment:
@0xdeadbee, one last thing. It confuses me that the request to create a contact requires [a comma-separated list](https://github.com/tyntec/api-collection/blob/master/sms/v1/openapi.yaml#L940) but the content type is [JSON](https://github.com/tyntec/api-collection/blob/master/sms/v1/openapi.yaml#L936). Shouldn't that be the `text/csv` content type?